Adyen.IntegrationTest/CheckoutTest.cs

```diff
@@ -69,10 +72,10 @@ public void PaymentsTest()
         {
             var paymentResponse = _checkout.Payments(CreatePaymentRequestCheckout());
             Assert.IsNotNull(paymentResponse.AdditionalData);
-            Assert.AreEqual("411111", paymentResponse.AdditionalData["cardBin"]);
```

Comment:
Noticed some rename-changes here, quick question, could you confirm this?

- [ ] cardBin is removed, renamed to cardSummary I assume?

- [ ] paymentMethodVariant is removed, renamed to paymentMethod I assume?

Reply:
Not too sure actually, as I'm seeing both in the specs under AdditionalCardData. Ditto for the paymentMethod, both still in the specs as well. But seeing as I don't encounter them anymore whilst debugging and getting the raw responses on the checkout IT's, I suspect they retired them (partly).